app/Models/LoanRepository.php

```
@@ -219,6 +239,13 @@ public function close(int $id): bool
         } catch (\Throwable $e) {
             SecureLogger::warning('LoanRepository::close flush notifications warning', ['error' => $e->getMessage()]);
         }
+        if ($reassignmentService !== null) {
+            try {
+                $reassignmentService->flushDeferredNotifications();
+            } catch (\Throwable $e) {
+                SecureLogger::warning('LoanRepository::close reassign flush warning', ['error' => $e->getMessage()]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'close'. (undefined)

(StaticAccess)